sources/zsh/history

```diff
@@ -1,2 +1,2 @@
 mount -o domain="$DOMAIN" -o username="$USER" -o password="$PASSWORD" -t cifs "//$DC_HOST/SYSVOL" /tmp/mnttarget/
-screen /dev/ttyACM0 115200
\ No newline at end of file
+screen /dev/ttyACM0 115200
```

Comment:
There has been a lot of change with the management of `history` and `aliases`. Now, history and aliases are separated by tools.
The folder `sources/zsh/history.d/` contains history files and the folder `sources/zsh/aliases.d/` contains aliases files.
If you want to add a `history command` as I recommended, you have to create a file with the name of the tool in the `history.d` folder.
You can take an example from : https://github.com/ThePorgs/Exegol-images/blob/main/sources/zsh/history.d/assetfinder.